test/Makefile

```diff
@@ -1,25 +1,50 @@
 DIR_MAKEFILE := ./make
+DIR_LOG      := ../log
 
-ifndef SYSTEM
-    ifeq ($(OS), Windows_NT)
-        SYSTEM := windows
-    else
-        SYSTEM := $(shell uname -s)
-
-        ifeq ($(filter $(SYSTEM),Linux Darwin),$(SYSTEM))
-            SYSTEM := unix_like
+ifeq (,$(wildcard $(DIR_LOG)/*.sys))
+    $(info There is no log including your operating system...)
```

Comment:
Replace msg with `Syslog file not found!`

Reply:
Done.